ontology/yaml/resources/HVAC/entity_types/FRZ.yaml

```diff
@@ -28,6 +28,15 @@ FRZ_DXZTC_DFSS:
   - DXZTC
   - DFSS
 
-
-
+FRZ_DFSS_DEFSS_DEFSTC:
+  description: "Freezer with discharge fan,compressor, defrost run and temperature control."
```

Comment:
you mention "compressor" but I do not see any compressor types here.

Reply:
Yes not included thank you for noticing added the required abstract type.